README.rst

```diff
@@ -86,3 +86,61 @@ This uses ``git`` to update the repository, fetches the latest build from docker
     :target: https://raw.githubusercontent.com/DigitalSlideArchive/NCI-SEER-Pediatric-WSI-Pilot/master/LICENSE
     :alt: License
 
+
+Workflow States and Transitions
+===============================
+
+There are several states an image can be in, including:
+
+ - imported
+ - quarantine
+ - processed
+ - rejected
+ - original
+ - finished
+
+Which correspond to named folders, i.e., an image will be in the ``imported`` state at the time it lives in the ``imported`` folder, as long as users move images between states using the DSA UI tools. By using other Girder admin tools, it is possible to break the correspondence between the state and the folder name, but that should be an exceptional and unusual case.
+
+The reason that there are named states that are separate from named folders is so that workflow provenance can be tracked. An image may currently be in the ``quarantine`` state in the ``quarantine`` folder, but the image's workflow history indicates that it had previously been in the ``imported`` state before the ``quarantine`` state.
+
+Import
+------
+
+When an image is first brought into the system, through the usual mechanism of putting the image plus metadata into an import filesystem folder and running an import script, the image is deposited into the ``import`` folder and is in the ``import`` state.
+
+TODO: describe structure for file/folder layout and csv file, which should go into import folder for data ingest
+
+Quarantine
```

Comment:
I think we should shift quarantine to after all the other states.

Specifically the work flow doesn't require the item to ever be quarantined -- items in Imported or Quarantine can be redacted.  I expect that the actual workflow is
```
                 
imported --redact--> processed --approve--> finished
                 \-> original  \--reject--> rejected
```
At any state, but typically if you have processed an image and realize more redacting could finish the job, you can click Quarantine to move to the quarantine state; this allows doing the redaction again.

So, in the processed state, the user will click "Finish" to approve it, "Reject" to mark that it is impossible to fix, of "Quarantine" for more reprocessing.

The currently available state transitions are:
Imported -> Process / Reject / Quarantine
Original -> Quarantine
Processed -> Finish / Quarantine
Rejected -> Quarantine
Quarantine -> Process / Reject / Unquarantine
Finish -> Quarantine

The original folder has the item before processing.  That is, when you process an item (redact it), a copy is made and placed in the Original folder for provenance.  The actual Girder item is modified (the file is redacted and the item's file is updated with this new version) and moved to the processed folder.